### DISCUSSION TITLE
discord的加速功能怎么消失了

### DISCUSSION THREAD

**Discussion by tou002** [BeyondDimension/SteamTools#3048] [General]
discord的加速功能怎么消失了
![屏幕截图 2023-11-28 182108](https://github.com/BeyondDimension/SteamTools/assets/102643572/56dd24a0-eac6-4d9f-aaa3-4a305a7a2f22)
今天是11月28号

**Comment by radehaba** [comment]
同问

**Reply by arrcojustin** [reply]
确实
突然消失

**Reply by arrcojustin** [reply]
上周我都用得好好的

**Reply by tou002** [reply]
我上周也用的好好的突然就没了UWP版也没有好像

**Reply by adoin** [reply]
说是被迫去掉了，只能社区维护带discord的了

**Reply by Team-no1** [reply]
怎么恢复discord的加速功能

**Reply by arrcojustin** [reply]
也至少告诉我们为啥关闭discord加速啊！

**Comment by elarbor** [comment]
vercel.app 的也不成功😂
![image](https://github.com/user-attachments/assets/e81f2144-d9a8-474e-ab28-8f2c5f2a8339)
![image](https://github.com/user-attachments/assets/022177de-237a-4071-9174-13d1cd50ac9d)
![image](https://github.com/user-attachments/assets/7e3d097b-4b57-4858-a691-9f429402cfac)
